app/assets/javascripts/application.js

```diff
@@ -213,7 +213,9 @@ function split(val) {
 
 // extract last term from a string of autocomplete entries
 function extractLast(term) {
-    return split(term).pop();
+    sanitizedTerm = term.trim(); // remove trailing whitespace to prevent returning all results
+    console.log('attempting to autocompelte on "' + sanitizedTerm + '"');
```

Comment:
If this `console.log` is indeed needed:

```suggestion
    console.log('attempting to autocomplete on "' + sanitizedTerm + '"');
```

Reply:
It's not, actually - that was for debugging purposes.  I'll remove it.

Reply:
64e4e5c